LINK/usr/bin/hourly_reindex_metrics_tables

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+source /etc/default/evm
+LOGFILE=/var/www/miq/vmdb/log/hourly_continuous_pg_maint_stdout.log
+TABLE_NAME=metrics_$(date -u +"%H" --date='+1 hours')
+
+echo "current time is $(date) -> target for REINDEX TABLE is '$TABLE_NAME' table" >> $LOGFILE
+psql -U postgres vmdb_production -a -e -c "REINDEX TABLE $TABLE_NAME" >> $LOGFILE 2>&1
```

Comment:
Yeah, I wasn't sure if I wanted a common file for DRYing some of this stuff up or if I should just combine all of the `hourly_reindex*` into a single file that takes an argument.

What do you think?